streams-bootstrap-cli/src/main/java/com/bakdata/kafka/streams/KafkaStreamsApplication.java

```diff
@@ -72,32 +70,29 @@
 public abstract class KafkaStreamsApplication<T extends StreamsApp> extends
         KafkaApplication<StreamsRunner, StreamsCleanUpRunner, StreamsExecutionOptions,
                 ExecutableStreamsApp<T>, ConfiguredStreamsApp<T>, StreamsTopicConfig, T, StreamsAppConfiguration> {
-    @CommandLine.Option(names = "--input-topics", description = "Input topics", split = ",")
-    private List<String> inputTopics = emptyList();
-    @CommandLine.Option(names = "--input-pattern", description = "Input pattern")
-    private Pattern inputPattern;
-    @CommandLine.Option(names = "--error-topic", description = "Error topic")
-    private String errorTopic;
-    @CommandLine.Option(names = "--labeled-input-topics", split = ",", description = "Additional labeled input topics",
-            converter = {UseDefaultConverter.class, StringListConverter.class})
-    private Map<String, List<String>> labeledInputTopics = emptyMap();
-    @CommandLine.Option(names = "--labeled-input-patterns", split = ",",
-            description = "Additional labeled input patterns")
-    private Map<String, Pattern> labeledInputPatterns = emptyMap();
+    @Mixin
+    @Delegate
+    private InputOptions inputOptions = new InputOptions();
+    @Mixin
+    @Delegate
+    private OutputOptions outputOptions = new OutputOptions();
+    @Mixin
+    @Delegate
+    private ErrorOptions errorOptions = new ErrorOptions();
```

Comment:
Please add error topic to this test https://github.com/bakdata/streams-bootstrap/blob/master/streams-bootstrap-cli/src/test/java/com/bakdata/kafka/streams/KafkaStreamsApplicationCliTest.java#L366 and add an equvialent test for producer app